sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -227,5 +227,11 @@ namespace Azure { namespace Core {
     // Private constructor. Use static methods to create an instance.
     explicit DateTime(Duration const& since1601) : m_since1601(since1601) {}
     Duration m_since1601;
+
+  public:
+    /**
+     * @brief Construct an instance of @DateTime.
+     */
+    DateTime() : m_since1601(0) {}
```

Comment:
Didn't we settle on `01-01-0001`, from the linked issue.

Reply:
It is a separate issue planned for after the new year.

Reply:
Even if we initialize it now with `-504911232000000000LL`, which means it is `0001-01-01`, it will be a value that you can't get a string of, can't construct via ints constructor, or can't parse from a string. Full support is needed.